subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -944,33 +1004,49 @@ static uint8_t ptc_calc(const struct lll_adv_iso *lll, uint32_t event_spacing,
 	return 0U;
 }
 
+static uint32_t adv_iso_time_get(struct ll_adv_iso_set *adv_iso, bool max)
+{
+	struct lll_adv_iso *lll_iso;
+	uint32_t ctrl_spacing;
+	uint32_t pdu_spacing;
+	uint32_t time_us;
+
+	lll_iso = &adv_iso->lll;
+
+	pdu_spacing = PDU_BIS_US(lll_iso->max_pdu, lll_iso->enc, lll_iso->phy,
+				 lll_iso->phy_flags) +
+		      EVENT_MSS_US;
+	ctrl_spacing = PDU_BIS_US(sizeof(struct pdu_big_ctrl), lll_iso->enc,
+				  lll_iso->phy, lll_iso->phy_flags);
+
+	if (IS_ENABLED(CONFIG_BT_CTLR_ADV_ISO_RESERVE_MAX) || max) {
+		time_us = (pdu_spacing * lll_iso->nse * lll_iso->num_bis) +
+			  ctrl_spacing;
+	} else {
+		time_us = pdu_spacing * ((lll_iso->nse * lll_iso->num_bis) -
+					 lll_iso->ptc);
+	}
+
+	time_us += EVENT_OVERHEAD_START_US + EVENT_OVERHEAD_END_US;
```

Comment:
Any chance that `time_us` can overflow? 

You are multiplying a `uint32_t` by another another number and storing the result in another `uint32_t`.

Reply:
`time_us` will not overflow for the Max PDU size of 251 bytes on S8 coding, with NSE of 31 and 31 BISes. Added comments in the code calculating the required bits for the maximum radio event time.